CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Change Log
 
+## [0.27.0](https://github.com/redhat-developer/vscode-xml/milestone/36?closed=1) (May 27, 2024)
```

Comment:
Should we release this as 0.28.0 so that lemminx and vscode-xml retain the same minor version?

Reply:
no need

Reply:
okay